src/clj/rems/api/services/invitation.clj

```diff
@@ -76,7 +76,7 @@
           {:success false
            :errors [{:key :t.accept-invitation.errors.already-member/workflow}]}
           (do
-            (workflow/edit-workflow! {:id (:workflow/id workflow)
+            (workflow/edit-workflow! {:id workflow-id
```

Comment:
Should we keep id references consistent? Since you refactored `user-id `to be `userid`, should we do the same to `workflow-id`?

Reply:
It's like this because `userid` is legacy and we'd want to remain as consistent as possible so that we can find all the uses. We would ideally have everything follow the `xxx-id` convention or `xxx/id`. The only exception so far is the user identity (that in the db is called `eppn` because of legacy reasons too.